queue_processors/autoreduction_processor/autoreduction_processor.py

```diff
@@ -136,24 +136,28 @@ class Consumer:
     """ Class used to setup the queue listener. """
     def __init__(self):
         """ Initialise consumer. """
+        # Note: The consumer name of autoreduction_processor.py is "queueProcessor"
+        #   while the consumer name of queue_processor.py is "Autoreduction_QueueProcessor".
+        #   This seems like it needs changing...
         self.consumer_name = "queueProcessor"
 
-    @staticmethod
-    def run():
+    def run(self):
         """
         Connect to ActiveMQ via the QueueClient and listen to the
         /ReductionPending queue for messages.
         """
+        # Note: To reviewer - I'm not very similar with subscriptions, consumers, etc.
+        #   I'm not 100% confident I've changed the code below
+        #   (from connection to client) correctly
         activemq_client = QueueClient()
-        connection = activemq_client.connect(listener=Listener())
-
-        # Create event listener
-
-        # connection.set_listener('Autoreduction', listener)
-        connection.subscribe(destination='/queue/ReductionPending',
-                             id='1',
-                             ack='auto',
-                             header={'activemq.prefetchSize': '1'})
+        activemq_client.connect()
+
+        activemq_client.subscribe_amq(consumer_name=self.consumer_name,
```

Comment:
I have had trouble with this function in the past. Could be the issue you're seeing?

Reply:
I had a suspicion this was the problem, so I replaced the code with the commented-out directly-using-stomp version below (which was the code before) - see 6db37df - but the Travis issue persisted..

I'm yet to continue my investigation of this issue, been working on other things this morning. I'll let you know after I've done so.